kickstarter/android-oss
# Fix: potential security vulnerability in one of our dependencies.

# 📲 What

- updated gem activesupport and fastlane plugin and the list of dependencies of those two gems.

# 🤔 Why
I hate having this warnings: 

![Screen Shot 2020-05-28 at 8 44 23 AM](https://user-images.githubusercontent.com/4083656/83163153-9065a680-a0bf-11ea-9d09-a4eaeb455e54.png)
![Screen Shot 2020-05-28 at 8 44 34 AM](https://user-images.githubusercontent.com/4083656/83163162-9491c400-a0bf-11ea-9a9a-4b551256b674.png)


# 🛠 How
Execute `bundle update activesupport`
Execute `bundle update fastlane`

# 📋 QA

- Execute `./bin/milkrun update-strings` -> you should get the translation without environment issues.
- Execute `bundle exec fastlane internal` -> you should have an internal release created & updated to firebase without any environment issues. Take a look into the date and hour to check is your latest internal release: 

![Screen Shot 2020-05-28 at 8 50 50 AM](https://user-images.githubusercontent.com/4083656/83163876-63fe5a00-a0c0-11ea-9a1c-d28a1084bd2c.png)
